build/package/Dockerfile

```diff
@@ -8,6 +8,7 @@ ENV OPENSSL_FORCE_FIPS_MODE=1
 WORKDIR /go/src/app
 COPY . .
 USER 0
+RUN git log -1
```

Comment:
The additional command logs the latest revision and ensures that the container has access to VCS information.